src/test/groovy/ratpack/zipkin/support/B3PropagationHeaders.groovy

```diff
@@ -0,0 +1,14 @@
+package ratpack.zipkin.support
+
+
+enum B3PropagationHeaders {
```

Comment:
There are constants for these here: https://github.com/openzipkin/brave/blob/master/brave/src/main/java/brave/propagation/B3Propagation.java#L22